common/app/common/cms/PlayCmsService.java

```diff
@@ -0,0 +1,38 @@
+package common.cms;
+
+import play.i18n.Lang;
+import play.i18n.Messages;
+import play.i18n.MessagesApi;
+import play.libs.F;
+
+import javax.inject.Inject;
+import javax.inject.Singleton;
+import java.util.Locale;
+
+/**
+ * Service that provides content data from Messages files as handled by Play.
+ * In order to share a common interface with other CMS, a normal message key has been split into Page Key and Message Key.
```

Comment:
We have `PlayCmsService.getPage(locale, pageKey)`, which returns a CmsPage, and then `PlayCmsPage.get(messageKey)`. I'll explain it better in the Javadoc.